examples/bq-remote-function/string_formatter/src/main/java/com/google/util/UserDefinedContext.java

```diff
@@ -0,0 +1,52 @@
+package com.google.util;
```

Comment:
Unless I'm mistaken, I don't see this class imported into any other class, can you please explain its use case and where it's needed ?

Reply:
yes, it is been used to define json properties.